src/main/java/math/Fraction.java

```diff
@@ -244,6 +244,9 @@ public long longValue() {
      */
     public boolean isZero() {
         // TODO (EGG6BH)
+        if (this.denominator == 0) {
```

Comment:
Note that the denominator is never zero, this is ensured by the constructor. Thus, this check should be removed.